task2/interface_manager.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+from collections import defaultdict
+from typing import List, Dict
+
+import requests
+from marshmallow import EXCLUDE
+
+from argument_parser import arg_pars
+from data_type import Interface
+from utils import get_logger
+
+
+def protocol(ssl):
+    return 'https'if ssl else 'http'
+
+
+class InterfaceManager:
+    def __init__(self, api_url, logger, args):
```

Comment:
@patrikprovaznik please use typehints here and everywhere where you have some parameters.. in some methods you are missing even typehint return type